### ISSUE TITLE
OMDb is no longer free

### ISSUE THREAD

**Issue by matt-richardson** [awslabs/aws-lex-convo-bot-example#2]
Started walking through the setup instructions, (which btw, dont match the lambda ui), I tested it and got the following:

```
START RequestId: 1a231f51-3d1b-11e7-9b61-9d8a066744b3 Version: $LATEST
2017-05-20T05:13:47.074Z	1a231f51-3d1b-11e7-9b61-9d8a066744b3	request received for userId=${intentRequest.userId}, intentName=${intentRequest.currentIntent.intentName}
2017-05-20T05:13:47.075Z	1a231f51-3d1b-11e7-9b61-9d8a066744b3	request received for Slots=Suicide Squad, Director
2017-05-20T05:13:47.581Z	1a231f51-3d1b-11e7-9b61-9d8a066744b3	[Error: status code: 401]
END RequestId: 1a231f51-3d1b-11e7-9b61-9d8a066744b3
REPORT RequestId: 1a231f51-3d1b-11e7-9b61-9d8a066744b3	Duration: 528.15 ms	Billed Duration: 600 ms 	Memory Size: 128 MB	Max Memory Used: 30 MB	
```

After fighting with IAM roles for ages trying to figure out why it wasn't allowing access, It appears that OMDb is [no longer free(https://www.patreon.com/posts/api-is-going-10743518), which means the example no longer works :(

**Comment by karangarg45** [comment]
yes getting the same error...any workaround?

**Comment by matt-richardson** [comment]
I ended up hardcoding it to return a chunk of json.

**Comment by herbertjkelly** [comment]
Yeah same issue have a look here for a fork where you can add your own API key... https://github.com/misterhat/omdb/pull/29/files

$1 a month for an API key not free but not bad...

**Comment by tbuckham** [comment]
@matt-richardson - did you just replace the API call to OmDB with another call to a different API out of interest? Was this is a quick update in the code? Thanks, Tom

**Comment by pubalisen** [comment]
Hey guys the code is updated to use TMDB instead. Please use this Lambda function.

https://s3-us-west-2.amazonaws.com/re-invent-botworkshop/samples/moviePedia/moviePedia.zip
